gbucket-snippets/copy_data_to_google_bucket.R

```diff
@@ -0,0 +1,27 @@
+# This snippet assumes that you run setup first
+
+# This code saves your dataframe into a csv file in a "data" folder in Google Bucket
+
+# Replace df with THE NAME OF YOUR DATAFRAME
+my_dataframe <- df
+
+# Replace 'test3.csv' with THE NAME of the file you're going to store in the bucket (don't delete the quotation marks)
+destination_filename <- 'test.csv'
+
+########################################################################
+##
+################# DON'T CHANGE FROM HERE ###############################
+##
+########################################################################
+
+# store the dataframe in current workspace
+write_excel_csv(my_dataframe, destination_filename)
+
+# Get the bucket name
+my_bucket <- Sys.getenv('WORKSPACE_BUCKET')
+
+# Copy the file from current workspace to the bucket
+system(paste0("gsutil cp ./", destination_filename, " ", my_bucket, "/data/"), intern=T)
```

Comment:
Are we always going to use the current working directory? I thought there was a dedicated local files directory somewhere.

Reply:
@jaycarlton  I'm not aware of a dedicated local files directory. Is this a new feature?  (e.g., you folks are changing the notebooks notion of the current working directory to _not_ be the directory in which the notebook resides?)

Reply:
Confirmed with @calbach that there's no official directory you should use. PWD is common practice, so no issues here. Thanks.